appInsight-api.js

```diff
@@ -0,0 +1,15 @@
+if (typeof process.env.APP_INSIGHT_CONNECTION_STRING !== "undefined") {
```

Comment:
the setup of this file isn't in the app insight docs, is it really required?

Reply:
It is because we have to provide that variable even before it loads .env file from the Next Server. So for local, we have to create .env file on the root folder to access the variable otherwise it will fail on both local and on the PR slot if that variable doesn't exist. We can also turn on the logging for testing purposes by adding the key in the app config and restarting the App.